### PR TITLE
Add first swarm provider use case guide

### PR DESCRIPTION
**Pull request by alexperezau**
First guide showing a generic use case to swarm 3 manager VMs.

### CODE REVIEW

**Comment by alexperezau** [comment]
Need to fix the provider url.

**Review by prologic — APPROVED** [review]

File: docs/guides/use_case_3_manager_swarm.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Swarm Provider Use Case - 3 node manager swarm
```
Comment (prologic) [inline docs/guides/use_case_3_manager_swarm.md:1]:
```suggestion
---
page_title: "Creating a 3-manager node Swarm Cluster with vSphere"
---

# Creating a 3-manager node Swarm Cluster with vSphere
```

**Review by prologic — APPROVED** [review]
Maybe a _slightly_ better title for this guide?

**Comment by prologic** [comment]
Maybe rename the doc to `3-manager-cluster-vsphere.md`?

**Pull request merged**